src/components/TodoComponent/index.js

```diff
@@ -1,14 +1,35 @@
 import React from "react";
+import axios from "axios";
 import { Button } from "@material-ui/core";
 import MenuIcon from "@material-ui/icons/Menu";
 import AddCircleOutlineOutlinedIcon from "@material-ui/icons/AddCircleOutlineOutlined";
 import "./index.css";
+import Lists from "./lists.js";
 
 class TodoComponent extends React.Component {
+  state = {
+    todoLists: [],
+    tasks: [],
+  };
+
   createNewList = () => {
     this.props.history.push("/new_list");
   };
 
+  componentDidMount = () => {
+    this.getTodoList();
+  };
+
+  getTodoList = async (res) => {
+    try {
+      const res = await axios.get("/api/todolist");
+      this.setState({ todoLists: res.data.lists });
```

Comment:
No need to write 2 setState. You need to combine them into one line:
this.setState({ todoLists: res.data.lists, tasks: res.data.tasks });